README.md

```diff
@@ -147,7 +168,7 @@ metadata:
   namespace: openshift-marketplace
 spec:
   sourceType: grpc
-  image: $IMAGE_REGISTRY/$REGISTRY_NAMESPACE/hco-container-registry:$CONTAINER_TAG
+  image: $IMAGE_REGISTRY/$REGISTRY_NAMESPACE/hyperconverged-cluster-index:1.10.0
```

Comment:
we exported `$CONTAINER_TAG` earlier in the instructions, why have the version here?

Reply:
Corrected.